public/transaction.js

```diff
@@ -1,14 +1,12 @@
-
-
 function transaction() {
-    return (
-        <>
-            
-            <a className="btn" href="/#/deposit" onClick={Deposit}>Deposit</a>
```

Comment:
If you want these buttons to redirect to your Deposit/Withdraw components, you only need the href or you could use a `<Link/>` from react-router.
Passing down the component to the onClick event handler, didn't make much sense here and was firing the error of "invalid hook call"